### ISSUE TITLE
NVDA 2023.3beta4 and add-on store:  The "updatable add-ons" tab is not announced when accessed for the first time by "control+tab".

### ISSUE THREAD

**Issue by paulber19** [nvaccess/nvda#15628]
-open the add-on store.
- after possibly responding with "OK" to the NVDA warning, the list of installed add-ons is focused.
- type "control+tab".
- The list of "updatable add-ons" is then focused
But NVDA doesn't announce it, but instead announces: "tab control panel"
This only happens when the list is first focused.

### Expected behavior:

NVDA must say: "Updatable add-ons tab selected"
#### NVDA installed/portable/running from source:
NVDA portable
#### NVDA version:
NVDA 2023.3beta4 and alpha
#### Windows version:
Windows 10 22H2 (10.0.19045) workstation x86

#### Name and version of other software in use when reproducing the issue:
None
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by CyrilleB79** [comment]
Duplicate of #14986.
Closing.
